filters/src/main/java/io/camunda/zeebe/process/test/filters/logger/RecordStreamLogger.java

```diff
@@ -272,7 +272,14 @@ private String logVariables(final Map<String, Object> variables) {
     }
 
     final StringJoiner joiner = new StringJoiner(", ", "[", "]");
-    variables.forEach((key, value) -> joiner.add(key + " -> " + value.toString()));
+    variables.forEach(
```

Comment:
:wrench: This could also be rewritten to (pseudo code)
```
variables.entries().stream()
   .map( (key, value) -> key + " -> " +  value == null ? "null" :  value )
   .collect(Collectors.joining(", "));
```

Reply:
Or even simpler:
` variables.forEach((key, value) -> joiner.add(key + " -> " + value()));`